src/Console/Domains/Commands/CommandHelpers.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Enflow\Component\Laravel\Console\Domains\Commands;

```

Comment:
Can place that helper directly in the Console namespace. Seperate namespace isn't really needed.